### PR TITLE
TriggerAim / MagnetTrigger

### PR DESCRIPTION
**Pull request by steel4me**
I know, it is ugly code, but it works and maybe someone of you can optimize it. Or someone can use it.
I think it is a good feature for gamers that use aimbot with aimlock and hold key on mouse 1. 
When TriggerAim is active with mouse 5 for example, trigger will activate aimbot and make single shots while aimlocking.
I love it ;)

I don't know why the pull request have some code pulled that **was not changed**...Sorry for that, im learning how to use github

### CODE REVIEW

File: Osiris/Hacks/Aimbot.cpp [hunk 1]
```diff
@@ -94,38 +94,46 @@ static bool canScan(Entity* localPlayer, Entity* entity, const Vector& destinati
...
 
-    if (!config.aimbot[weaponIndex].enabled)
-        weaponIndex = 0;
+	if (!config.aimbot[weaponIndex].enabled)
+		weaponIndex = 0;
 
-    if (!config.aimbot[weaponIndex].betweenShots && activeWeapon->nextPrimaryAttack() > memory.globalVars->serverTime())
-        return;
+	if (!config.aimbot[weaponIndex].betweenShots && activeWeapon->nextPrimaryAttack() > memory.globalVars->serverTime())
+		return;
```
Comment (steel4me) [inline Osiris/Hacks/Aimbot.cpp]: no changes here, i think it was automaticly caused by visual studio while compiling somehow...

**Comment by thomas0aotian** [comment]
so do aimbot & trigaim share same cfg? or its individual

**Comment by steel4me** [comment]
they share the same, when you watch the changes, it's just a "keymapping"... quick and dirty ;)

**Comment by thomas0aotian** [comment]
Better one here #751

**Comment by steel4me** [comment]
THX @thomas0aotian will implement your way soon.

**Comment by thomas0aotian** [comment]
Alright bud and u sneakly implement standalone rcs u nerd. :smile: about to try it tmr, better be working:kissing_heart:

**Pull request closed without merging**